oci-ignite-advanced/create-adw/autonomous-data-warehouse.md

```diff
@@ -0,0 +1,219 @@
+    - __Connection Type__ - For this lab, specify __Cloud Wallet__ (if you are using SQL Developer 18.2, this is **Cloud PDB**). Browse the location where you saved the *wallet.zip* in __*Step 5*__ and **select the .zip file**.
+    - __Service__ - Enter the database TNS name. The client credentials file includes a *tnsnames.ora* file that provides database TNS names with corresponding services.
+    - __Auto Scaling__ - For this lab, keep auto-scaling enabled, to enable the system to automatically use up to three times more CPU and IO resources to meet workload demand.
+    
+    Click **Test** to test the database connection. If *Status:Success* then click **Connect** to connect to the database.
+
+    ![Configure the database connection parameters.](images/configure-connection-sqldeveloper.png " ")
+
+## Task 4: Run a Sample Query on the Autonomous Database instance via SQL Developer
+
+
+1. Click on the **"+"** next to the database connection *IGNITEADW* on the left side to expand the database content and start testing it.
+
+    ![Connection succesful](images/connection-successful-sqldeveloper.png " ")
+
+2. Click on the **Copy** button below to copy **Sample Sales History Data Set** Query and **Paste** it into the SQL Developer worksheet:
+
+    ```
+    <copy>
+SELECT channel_desc, TO_CHAR(SUM(amount_sold),'9,999,999,999') SALES$,
+   RANK() OVER (ORDER BY SUM(amount_sold)) AS default_rank,
+   RANK() OVER (ORDER BY SUM(amount_sold) DESC NULLS LAST) AS custom_rank
+FROM sh.sales, sh.products, sh.customers, sh.times, sh.channels, sh.countries
+WHERE sales.prod_id=products.prod_id AND sales.cust_id=customers.cust_id
+  AND customers.country_id = countries.country_id AND sales.time_id=times.time_id
+  AND sales.channel_id=channels.channel_id
+  AND times.calendar_month_desc IN ('2000-09', '2000-10')
+  AND country_iso_code='US'
+GROUP BY channel_desc;
+    ```
+3. Click the green **"Play"** button to run the statement.
+![Paste the sample Query in the SQL worksheet and Run it](images/sample-query-sqldeveloper.png " ")
+
+4. Check the **Query Result**.
+
+![Sample Query Result](images/sample-query-result-sqldeveloper.png " ")
+
+
+    
+
+See this [tutorial](https://www.oracle.com/webfolder/technetwork/tutorials/obe/cloud/adwc/OBE_Running%20a%20Query%20on%20Sample%20Data/running_a_query_on_sample_data.html#QuerytheSampleStarSchemaBenchmarkDataSet) to test the performance of different Autonomous Data Warehouse configurations.
+
+
+Please *proceed to the next lab*.
+
+## Learn more
+
+See the [documentation](https://docs.oracle.com/en/cloud/paas/autonomous-data-warehouse-cloud/user/autonomous-workflow.html#GUID-5780368D-6D40-475C-8DEB-DBA14BA675C3) on the typical workflow for using Autonomous Data Warehouse.
+
+
+## Acknowledgements
+*Congratulations! You have successfully completed the lab.*
```

Comment:
Delete this line here or move it about the Learn more section

Reply:
done.